src/notifications/contracts/mongo-database-example.js

```diff
@@ -4,12 +4,16 @@ import NotificationDatabaseContract from './database';
 import Notification from './mongo-model-example';
 
 class MongoNotificationDatabase extends NotificationDatabaseContract {
+  constructor() {
```

Comment:
Not needed, ``NotificationDatabaseContract`` works as an interface, it's only needed if you need to do something in the constructor (which you aren't) so adding the constructor with super doesn't make sense in here IMHO.